### DISCUSSION TITLE
awk: command not found

### DISCUSSION THREAD

**Discussion by zillurbmb51** [nextflow-io/nextflow#3846] [Q&A]
Hi,
I have a script with multiple processes. In one process, I used `awk` and now I am getting error:
```
command exit status 127
awk: command not found.
```
Any help? Attached is the log file.
[nextflow.log4.txt](https://github.com/nextflow-io/nextflow/files/11178050/nextflow.log4.txt)

**Comment by bentsherman** [comment]
Depends on the environment where the task is running. Is it a local task? Is it a container? Try to run `awk` in that environment and see if it's available.
